src/data/markdown/translated-guides/en/02 Using k6/12 Execution context variables.md

```diff
@@ -31,24 +31,30 @@ $ k6 run --vus 10 --iterations 100 script.js
 
 ## \_\_VU and \_\_ITER
 
-**\_\_VU** and **\_\_ITER** are both global variables with execution context information that k6 makes
-available to the test script.
+**\_\_VU** and **\_\_ITER** are both global variables with execution context information that k6 makes available to the test script.
 
 ### \_\_ITER
 
 A numeric counter with the current iteration number for a specific VU. Zero-based.
 
 ### \_\_VU
 
-Current VU number. The value is assigned incrementally for each new VU instance. One-based.
-However, VU number is 0 while executing the setup and teardown functions.
+Current VU number in use. The value is assigned incrementally for each new VU instance, starting from one. The variable will be 0 while executing the setup and teardown functions.
+
+### ⚠️ To note when running in the k6 Cloud
+
+When you run your tests in the [k6 Cloud](/cloud) you will have additional [environment variables](/using-k6/environment-variables) that will tell you on which server, load zone and distribution of the test you are currently executing. You can read more about them [here](/using-k6/environment-variables). 
```

Comment:
The link about the additional variables is wrong -> https://k6.io/docs/cloud/creating-and-running-a-test/cloud-tests-from-the-cli/#environment-variables